Nodejs/Product/Nodejs/Project/ImportWizard/ImportSettings.cs

```diff
@@ -392,13 +389,7 @@ ProjectLanguage projectLanguage
 
             if (typeScriptSupport)
             {
-                writer.WriteElementString("TypeScriptSourceMap", "true");
-                writer.WriteElementString("TypeScriptModuleKind", "CommonJS");
                 writer.WriteElementString("EnableTypeScript", "true");
-                if (typeScriptVersion != null)
-                {
-                    writer.WriteElementString("TypeScriptToolsVersion", typeScriptVersion);
```

Comment:
Are we giving up the ability to fix people on a particular TS version?

Reply:
Only for SDK, if the user wants a specific version of TS they need to install it through NPM since we're using whatever version is installed in node_modules. e.g. `npm install --save-dev typescript@3.0.1`